### DISCUSSION TITLE
imgui_widgets.cpp gives 200+ errors

### DISCUSSION THREAD

**Discussion by mentalbreakdown-cplusplus** [ocornut/imgui#7981] [New Users Build/Link/Run issues ONLY!]
### Version/Branch of Dear ImGui:

Version 1.91.2

### Back-ends:

imgui_widgets.cpp

### Compiler, OS:

windows 10

### Full config/build information:

dear imgui 1.91.2 (imgui_widgets.cpp) 

### Details:

**why does imgui_widgets.cpp give me 200+ errors?:**

im using vs 2022 and im making a 3D freeglut based game engine, when including imgui and trying to debug im getting more than 200 errors from the file "imgui_widgets" and other imgui files like imgui.cpp. First i thought that vs is bugged but i restarted my entire pc and that did nothing so i checked the dirs and still nothing.

### Minimal, Complete and Verifiable Example code:

```cpp
// Here's some code anyone can copy and paste to reproduce your issue
ImGui::Begin("Example Bug");
MoreCodeToExplainMyIssue();
ImGui::End();
```

**Comment by cfillion** [comment]
What are the errors you're getting?
